### PR TITLE
adding LFS171x sawtooth code to education folder

### PR DESCRIPTION
**Pull request by alexandrag2254**
Signed-off-by: Alexandra Groetsema <alexandrag2254@gmail.com>

### CODE REVIEW

File: LFS171x/sawtooth-material/sawtooth-tuna/client/index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/client/index.html]: Can we add a comment to this file to specify the license using SPDX short identifiers?

File: LFS171x/sawtooth-material/sawtooth-tuna/client/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "Sawtooth-tuna-chain-client",
+  "version": "0.0.0",
+  "description": "A simple browser based blockchain client",
+  "main": "",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/LFS171x/sawtooth-material/sawtooth-tuna.git"
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/client/package.json]: This is missing the education folder. Also, probably needs to be: https://github.com/hyperledger/education.git

File: LFS171x/sawtooth-material/sawtooth-tuna/client/src/app.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
+'use strict'
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/client/src/app.js]: Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0

File: LFS171x/sawtooth-material/sawtooth-tuna/client/src/components.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+'use strict'
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/client/src/components.js]: Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`

File: LFS171x/sawtooth-material/sawtooth-tuna/client/src/state.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
+'use strict'
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/client/src/state.js]: Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`

File: LFS171x/sawtooth-material/sawtooth-tuna/client/webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+const path = require('path')
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/client/webpack.config.js]: Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`

File: LFS171x/sawtooth-material/sawtooth-tuna/processor/handlers.js [hunk 7]
```diff
@@ -0,0 +1,140 @@
+'use strict'
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/processor/handlers.js]: Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`

File: LFS171x/sawtooth-material/sawtooth-tuna/processor/index.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+'use strict'
```
Comment (tkuhrt) [inline LFS171x/sawtooth-material/sawtooth-tuna/processor/index.js]: Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`

File: LFS171x/sawtooth-material/sawtooth-tuna/client/index.html (hunk 1 above)
Comment (alexandrag2254) [reply LFS171x/sawtooth-material/sawtooth-tuna/client/index.html]: Sure! At the very top? Good to add this comment to the Fabric index.html page as well?

Comment (tkuhrt) [reply LFS171x/sawtooth-material/sawtooth-tuna/client/index.html]: Yes, we definitely want this on the Fabric work too.

**Pull request merged**